_drafts/blog-post-1.md

```diff
@@ -0,0 +1,18 @@
+# Do *you* know where your data goes?
```

Comment:
Have a look at the [sample post in the drafts folder](https://github.com/cdl-geneseo/cdl-geneseo.github.io/blob/master/_drafts/sample_post.md) to see how to put your post title in the yaml header of your post (together with other metadata). Be sure to click the "Raw" button to see what the yaml header looks like in markdown.